src/ui/chat-widget/views/ChatWidget.css

```
@@ -0,0 +1,30 @@
+  inset-inline-end: 20px;
+}
+
+.ai-chatbot .ai-chatbot__brand {
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.ai-chatbot .pf-chatbot__message-and-actions {
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)